noir-projects/aztec-nr/README.md

```diff
@@ -1,5 +1,5 @@
 <div align="center">
-  <img height="210x" src="./assets/Aztec_banner.png" />
+  <img height="210" src="./assets/Aztec_banner.png" />
```

Comment:
This surely meant to be 210px?

Reply:
![image](https://github.com/user-attachments/assets/cdd34ea1-7452-474e-a3a7-22f31f80be9b)
According to the documentation, the `height` attribute should be specified as an integer without any units. You can find more information [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#height).